tests/test_kernel32/test_handle.py

```diff
@@ -120,3 +122,95 @@ def test_process_dies_before_timeout(self):
         self.assertEqual(
             WaitForSingleObject(hProcess, library.INFINITE),
             library.WAIT_OBJECT_0)
+
+
+class TestGetHandleInformation(TestCase):
+    """
+    Tests for :func:`pywincffi.kernel32.GetHandleInformation`
+    """
+    def _expected_inheritance(self):
+        # Python >= 3.4 creates non-inheritable handles (PEP 0446)
+        if sys.hexversion >= 0x30400f0:
+            return 0
+        else:
+            return 1
+
+    def test_get_handle_info_stdin(self):
```

Comment:
This test is failing on Python 3.4:

https://ci.appveyor.com/project/opalmer/pywincffi/build/726